exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/aws/AwsRequestSigningApacheInterceptorTest.java

```diff
@@ -39,13 +40,13 @@
 import software.amazon.awssdk.auth.signer.Aws4Signer;
 import software.amazon.awssdk.regions.Region;
 
-class AwsSignHttpRequestInterceptorTest {
+class AwsRequestSigningApacheInterceptorTest {
```

Comment:
❓ Do you think it makes sense to keep this test?

Reply:
We are testing that the library does what we are expecting, to me make sense. For future update/breaking change of the library have a test that ensure the expected behaviour it will be usefull

Reply:
Usually I'd say it doesn't make sense to unit test code of a library we use. We don't do this for the other libraries either. Instead, we should have an integration test that verifies if we can export when AWS is enabled.

The library itself should be responsible to test their own interceptor, [which they do](https://github.com/acm19/aws-request-signing-apache-interceptor/blob/master/src/test/java/io/github/acm19/aws/interceptor/http/AwsRequestSigningApacheInterceptorTest.java).

However, I think this is a special case. I can't find a way to write an IT where request signing is required. We can start an OpenSearch testcontainer and enable AWS, but this won't achieve anything as it's not running in AWS so it doesn't expect the request to be signed. As we do want verification this might be the best we can do for now.

Reply:
I'm agree with you!